sp800-63a/sec1_2_introduction.md

```diff
@@ -37,23 +37,21 @@ The following table states which sections of the document are normative and whic
 
-**IAL2** - Evidence supports the real-world existence of the claimed identity and verifies that the applicant is appropriately associated with this real-world identity. IAL2 introduces the need for either remote or physically-present identity proofing. Attributes could be asserted by CSPs to RPs in support of pseudonymous identity with verified attributes. A CSP that supports IAL2 can support IAL1 transactions if the user consents.
+**IAL2** - Evidence supports the real-life existence of the claimed identity and verifies that the applicant is appropriately associated with this real-life identity. IAL2 introduces the need for either remote or physically-present identity proofing. Attributes could be asserted by CSPs to RPs in support of pseudonymous identity with verified attributes. A CSP that supports IAL2 can support IAL1 transactions if the user consents.
 
 **IAL3** - Physical presence is required for identity proofing. Identifying attributes must be verified by an authorized and trained representative of the CSP. As with IAL2, attributes could be asserted by CSPs to RPs in support of pseudonymous identity with verified attributes.  A CSP that supports IAL3 can support IAL1 and IAL2 identity attributes if the user consents.
 
-At IAL2 and IAL3, pseudonymity in federated environments is enabled by limiting the number of attributes sent from the CSP to the RP, or the way they are presented. For example, if a RP needs a valid birthdate but no other personal details, the RP should leverage a CSP to request just the birthdate of the subscriber. It is preferred for the RP to ask the CSP for an attribute reference. For example, if a RP needs to know if a claimant is older than 18 they should request a Boolean value, not the entire birthdate in order to evaluate age. Conversely, it may be beneficial to the user that uses a high assurance CSP for transactions at lower assurance levels.  For example, a user may maintain an IAL3 identity, yet should be able to use their CSP for IAL2 and IAL1 transactions.
+At IAL2 and IAL3, pseudonymity in federated environments is enabled by limiting the number of attributes sent from the CSP to the RP, or the way they are presented. For example, if a RP needs a valid birthdate but no other personal details, the RP should leverage a CSP to request just the birthdate of the subscriber. It is preferred for the RP to ask the CSP for an attribute reference. For example, if a RP needs to know if a subscriber is older than 18 they should request a Boolean value, not the entire birthdate in order to evaluate age. Users who maintain and IAL3 identity should be able to use their CSP for lower level and pseudonymous transactions, giving them the added benefit of being able to assert already verified attributes as part of these transactions. It should be noted, that since the individual will have undergone an identity proofing process at enrollment, interactions with the CSP may not necessarily be pseudonymous.
```

Comment:
lower level --> lower-level
already verified --> already-verified
No comma after "noted", or perhaps just get rid of "It should be noted that"